Makefile

```diff
@@ -48,35 +48,39 @@ run/e2e-tests: deps/stop
 
 .PHONY: build/worker
 build/worker:
-	@rm -f ./bin/worker
+	@rm -f ./bin/worker-*
```

Comment:
Performing an `rm` with a wildcard that doesn't match any file, returns an error similar to this:
```
No matches for wildcard './bin/worker-*'. See `help expand`.
```

To not break, we can change the command to always exit with 0, meaning it won't break the target. It can be done by adding a `|| true` at the end, having something like `@rm -f ./bin/worker-* || true`.